core/src/main/java/org/opensearch/sql/planner/logical/LogicalTableFunction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.planner.logical;
+
+import com.google.common.collect.ImmutableList;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.storage.Table;
+
+/**
+ * Logical Table Functions...Function which outputs table.
+ */
+@ToString
+@EqualsAndHashCode(callSuper = true)
+public class LogicalTableFunction extends LogicalPlan {
```

Comment:
What is the difference with Relation? Or it should extend Relation?

Reply:
FYI: https://github.com/crate/crate/blob/de1d721e810a7b03486ef7d7a753bf72c70ce8c3/server/src/main/java/io/crate/analyze/relations/TableFunctionRelation.java